README.md

```diff
@@ -10,6 +10,16 @@ Please check [this](src/main/resources/README.adoc) document for more detailed i
 
 # Version Notes
 
+## Version notes 1.7.9
+- adds capability of loading Launch4j native external configuration file
+```xml
+<configuration> 
+    <infile>${project.basedir}/etc/bin/config.xml</infile>
+ </configuration>
+```
+By default it will take from ${project.basedir}/src/main/launch4j/${project.artifactId}-launch4j.xml
```

Comment:
It's better to use `src/main/resources` as files from that location are automatically [filtered](http://maven.apache.org/plugins/maven-resources-plugin/examples/filter.html) and there is no need for additional setup to do it and such approach will follow Maven's convention